src/main/java/HM08/Clinic.java

```diff
@@ -0,0 +1,41 @@
+package HM08;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.concurrent.BlockingQueue;
+
+public class Clinic implements Runnable {
+
+    private BlockingQueue<Animal> animals;
+    static volatile boolean close = false;
+    private LocalDateTime timeToClose = LocalDate.now().atTime(18, 0); //For tests you need to change the time
+
+
+    public Clinic(BlockingQueue<Animal> animals) {
+        this.animals = animals;
+    }
+
+    @Override
+    public void run() {
+        while (!close) {
```

Comment:
If you use this type of logic use Thread.sleep or eqvivalent methods. This is code (original) kill perfomance of application